lib/cb2/breaker.rb

```diff
@@ -4,6 +4,7 @@ class CB2::Breaker
   def initialize(options)
     @service  = options[:service] || "default"
     @strategy = initialize_strategy(options)
+    @whitelist = if options[:whitelist] then options[:whitelist] else [] end
```

Comment:
i would try to use a more specific variable name. @processable_exceptions or @exceptions_whitelist or something of the sort.